tests/jerry/generators/generate-arguments-2.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+
+file = open("tests/jerry/arguments-2.js", "w")
```

Comment:
When opening/reading/writing a file, it is recommended to use the following syntax:
```python
with open("...", "w") as file:
    # indented by 4 space
    ... # other code
```

Another thing is for debugging purposes it would be good to write out the name of the script which generated the js file directly into the js file as a comment. This would make tracing back any issue to this generator faster.

Reply:
Your first observation is added, second will also be added soon

Reply:
Second also added